README.md

```diff
@@ -1,1134 +1,233 @@
+- Look at the `project/plugins.sbt` of your generated project and add the required plugins and resolvers there
+- Do the same for `build.sbt`
+- Put a Swagger specification with a ```.yaml``` or ```.json``` extension into the ```conf``` directory
+- Add a specification link (`->`) to the Play's routes file
 
-## Run Your Application
+### Running Your Application with the Plugin
 
-Before we go any further, let's run the application.
+Now let's run your application with the plugin:
 
 - Open a shell and `cd` into your service project directory.
-- Start `sbt` and `run` the service.
+- Start `sbt` and `run` the service. This creates a folder on your local machine, using the Activator template.
 - View the running application at [http://localhost:9000](http://localhost:9000).
 
-The service template comes with the Swagger UI frontend included, 
-run statically from the within Play, which provides a sandbox for your service. 
-The template is configured with a template Swagger API definition called `example.yaml` 
-and located in the `conf` directory of the Play application. 
+A single specification defines a single API; in our case, these are three API endpoints:
 
-The `example.yaml` definition provides an example [API description](https://github.com/zalando/play-swagger-service/blob/master/conf/example.yaml).
-
-This definition contains three end points: 
-- the `/token` path, which accept the `GET` and `POST` methods
-- the `/todos/{user_id}`, which accepts the `GET` method. 
-
-The `GET /token` API plays a role of an authentication server and is used by the Swagger UI for OAuth token requests.
-The `POST /token` API represents an authorization server and is used by the security part of the 
-generated code to validate OAuth tokens.
- 
-The `GET /todos/{user_id}` takes a path parameter `user_id` and returns a TODO list for given user. 
-For the client to be allowed to access this endpoint, it must provide an OAuth token with the scope `admin:org`. 
-The token can be requested using the Swagger UI.
-
-Try it out for yourself: 
-
-Click the [default](http://localhost:9000/) button to expand the API definition in the Swagger UI.
+- The `GET /token` API plays a role of an authentication server and is used by the Swagger UI for OAuth token requests. 
+- The `POST /token` API represents an authorization server and is used by the security part of the generated code to validate OAuth tokens.
+- The `GET /todos/{user_id}` takes a path parameter `user_id` and returns a TODO list for the given user. Use `security/example.yaml.scala`, the marshaller for OAuth2 tokens, to request an OAuth token with the scope `admin:org`. This will grant permission for the client to access this endpoint and enable you to test the API.  
 
+Click the default button to expand the API definition in the Swagger UI. Now you can change the specification or write some backend code and use the Swagger UI to see the results.
```

Comment:
`Now you can change the specification` -  this reads like it can be done using the Swagger UI which is not.